dwdatareader/__init__.py

```diff
@@ -96,6 +97,41 @@ def description(self):
     def number_of_samples(self):
         return DLL.DWGetScaledSamplesCount(self.index)
 
+    @staticmethod
+    def _c_int():
+        return ctypes.c_int(ctypes.sizeof(ctypes.c_int))
+
+    def _chan_prop_contents(self, chan_prop):
```

Comment:
Maybe rename this _chan_prop_int() since it returns int, similar to _chan_prop_str()?
I think it will be simpler coding if your main variable is ctype.c_int and passed byref rather than creating a small string buffer and treating it as a c_int.

Reply:
So true.